src/main/java/racing/CarGenerator.java

```diff
@@ -0,0 +1,17 @@
+package racing;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CarGenerator {
+
+    private CarGenerator() {
+    }
+
+    public static List<Car> createCars(String carNames) {
+        return Arrays.stream(carNames.split(","))
```

Comment:
```suggestion
        return Arrays.stream(carNames.split(SPLITE_DELIMITER))
```
매직넘버를 상수로 치환해 의미를 명확하게 해보는 것은 어떨까요?

Reply:
헉 매직넘버를 단순히 숫자 데이터에서만 생각했는데.. 이런 것도 다 해당이 되겠네요!!ㅎㅎ 피드백 감사합니다.